pkg/server/options/authorization.go

```diff
@@ -100,22 +100,38 @@ func (s *Authorization) ApplyTo(config *genericapiserver.Config, informer kcpkub
 		authorizers = append(authorizers, a)
 	}
 
-	// kcp authorizers
+	// kcp authorizers, these are evaluated in reverse order
+	// TODO: link the markdown
+
+	// bootstrap rules defined once for every workspace
 	bootstrapAuth, bootstrapRules := authz.NewBootstrapPolicyAuthorizer(informer)
 	bootstrapAuth = authz.NewDecorator("bootstrap.authorization.kcp.io", bootstrapAuth).AddAuditLogging().AddAnonymization().AddReasonAnnotation()
 
+	// resolves RBAC resources in the workspace
 	localAuth, localResolver := authz.NewLocalAuthorizer(informer)
 	localAuth = authz.NewDecorator("local.authorization.kcp.io", localAuth).AddAuditLogging().AddAnonymization().AddReasonAnnotation()
 
+	// everything below - skipped for Deep SAR
```

Comment:
@s-urbaniak this shouldn't be the case. maxPermissionPolicyAuth and systemCRDAuth should both be applied for deep authz.

Reply:
hrmpf, sorry, explained that wrong, my bad, will follow-up with a PR tomorrow 👍

Reply:
skipping max perm authorizer seems to be necessary to fix https://github.com/kcp-dev/kcp/issues/2384 (fixed in https://github.com/kcp-dev/kcp/pull/2385). what are we missing?